### PR TITLE
(maint) Remove unused code and reformat slightly

### PR DESCRIPTION
**Pull request by melissa**
This commit removes a patch file that is not applied and slightly
reorganized the component files to be more in line with current
standard component layout.

### CODE REVIEW

**Comment by McdonaldSeanp** [comment]
👍 I like it

**Comment by melissa** [comment]
WAIT! It's broken

File: configs/components/razor-server.rb [hunk 1]
```diff
@@ -9,9 +9,6 @@
     pkg.requires "libarchive-dev"
   end
 
-  pkg.directory File.join(settings[:install_root], "var", "razor")
```
Comment (McdonaldSeanp) [inline configs/components/razor-server.rb]: Is this missing now?

File: configs/projects/razor-server.rb [hunk 2]
```diff
@@ -45,6 +45,7 @@
   proj.directory proj.torquebox_prefix
   proj.directory proj.sysconfdir
   proj.directory proj.logdir
+  proj.directory proj.rundir
```
Comment (McdonaldSeanp) [inline configs/projects/razor-server.rb]: Is it no longer required for this to have owner: 'razor', group: 'razor' ?

File: configs/components/razor-server.rb (hunk 1 above)
Comment (melissa) [inline configs/components/razor-server.rb]: Nope, that directory should be created when the razor user is created. The breakage has to do with the ordering. I was trying to install the service files after I moved everything in that directory out of place, so it was failing to find the service files

File: configs/projects/razor-server.rb (hunk 2 above)
Comment (melissa) [inline configs/projects/razor-server.rb]: No, that's taken care of in a postinstall script

**Comment by melissa** [comment]
Okay, I think I un-broke it.

**Comment by McdonaldSeanp** [comment]
Everything looks 👍 to me (again)

**Comment by mckern** [comment]
@melissa I'm willing to merge this, if you're reasonably sure it's not broken anymore.

**Comment by melissa** [comment]
definitely no longer broken :)

**Pull request merged**